src/main/java/frc/robot/RobotContainer.java

```diff
@@ -177,46 +213,63 @@ private void initCommands() {
   // it to a {@link
   // edu.wpi.first.wpilibj2.command.button.JoystickButton}.
   private void configureButtonBindings() {
+    controllerCheck();
 
     // Intake
     if (intakeForwardCommand != null && intakeReverseCommand != null) {
       buttons[Constants.RBumper].whileHeld(intakeForwardCommand);
```

Comment:
Im not sure which direction intakeForward goes. Does it intake the cargo or expel the cargo?
From our controls meeting right bumper was expel cargo, right trigger was intake cargo.

Reply:
intakeForward takes the ball in, so I inversed it